client/player.js

```diff
@@ -26,7 +26,7 @@ var Player = function (position_, world_) {
     name: "scout",
     reloadTime:100,
     loadStatus:100,
-    magazine:300,
+    magazine:50,
```

Comment:
the server immediately sets the player's loadout when they join so these numbers don't matter

Reply:
From a previous change, reverted.